### PR TITLE
fix(db): add missing primary keys

### LINKED ISSUE #6451: [Bug][InnoDB] Setting up InnoDB cluster as Devlake DB
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar issues.

### What happened

Hi all , we tried configuring InnoDB cluster as Devlake DB. Met with an issue , few tables dont have primary keys in them . Group Replication has a requirement to have primary key in them .

Error shown - 
```
ERROR: The following tables do not have a Primary Key or equivalent column: 
lake._devlake_locking_stub, lake._tool_teambition_connections, lake._tool_bitbucket_pipeline_steps

Group Replication requires tables to use InnoDB and have a PRIMARY KEY or PRIMARY KEY Equivalent (non-null unique key). Tables that do not follow these requirements will be readable but not updateable when used with Group Replication. If your applications make updates (INSERT, UPDATE or DELETE) to these tables, ensure they use the InnoDB storage engine and have a PRIMARY KEY or PRIMARY KEY Equivalent.
If you can't change the tables structure to include an extra visible key to be used as PRIMARY KEY, you can make use of the INVISIBLE COLUMN feature available since 8.0.23: https://dev.mysql.com/doc/refman/8.0/en/invisible-columns.html

Checking instance configuration...

{
    "status": "error"
}
```

### What do you expect to happen

InnoDB cluster should have recognised devlake mysql instances .

### How to reproduce

1. Deploy devlake with a single mysql instance.
2. Deploy mysql-operator using helm-chart. ([link](https://dev.mysql.com/doc/mysql-operator/en/mysql-operator-installation-helm.html))
3. Deploy InnoDB cluster using helm-chart. ([link](https://dev.mysql.com/doc/mysql-operator/en/mysql-operator-innodbcluster-simple-helm.html))
4. Try validating mysql instance of step-1 for innodb cluster .(Used - dba.checkInstanceConfiguration('Mysql instance'))

### Anything else

We can bypass this issue by adding primary key to these tables for now .But it may culminate to become bigger problem when all datas are there on devlake and we try to upgrade Devlake to a higher version .

### Version

v0.19.0-beta2

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by d4x1**
### ⚠️ Pre Checklist

> Please complete _ALL_ items in this checklist, and remove before submitting

- [x] I have read through the [Contributing Documentation](https://devlake.apache.org/community/).
- [x] I have added relevant tests.
- [x] I have added relevant documentation.
- [x] I will add labels to the PR, such as `pr-type/bug-fix`, `pr-type/feature-development`, etc.

### Summary
What does this PR do?
Add missing primary keys.

### Does this close any open issues?
Closes #6451

### Screenshots
Include any relevant screenshots here.
##### Before
image

##### After
image

##### Before
image

##### After
image

##### Before
image

##### After
image

### Other Information
Any other information that is important to this PR.

Because this bug https://github.com/go-gorm/gorm/issues/5437,  old tables should be dropped first.

### CODE REVIEW

File: backend/server/services/locking.go [hunk 1]
```diff
@@ -44,11 +44,12 @@ func lockDatabase() {
 	}
 	errors.Must(db.Create(lockingHistory))
 	// 2. obtain the lock: using a never released transaction
-	// This prevent multiple devlake instances from sharing the same database by locking the migration history table
+	// This prevents multiple devlake instances from sharing the same database by locking the migration history table
 	// However, it would not work if any older devlake instances were already using the database.
 	lockingTx = db.Begin()
 	c := make(chan bool, 1)
 	go func() {
+		errors.Must(lockingTx.DropTables(models.LockingStub{}.TableName()))
```
Comment (abeizn) [inline backend/server/services/locking.go:52]: Why should we delete this table?

**Review by klesh — DISMISSED** [review]

**Comment by klesh** [comment]
Did you test it on a brand new Innodb instance?

**Review by klesh — APPROVED** [review]

File: backend/server/services/locking.go (hunk 1 above)
Comment (d4x1) [reply backend/server/services/locking.go:52]: Because primary keys cannot be `AutoMigrate` if old table exists.

**Comment by d4x1** [comment]
> Did you test it on a brand new Innodb instance?

I have test it on my standalone innodb based MySQL. I have no innodb cluster, so this PR just fix #6451  theretically.

**Pull request merged**